src/java/picard/util/TabbedTextFileWithHeaderParser.java

```diff
@@ -145,6 +149,8 @@ public boolean hasColumn(final String columnLabel) {
     public Set<String> columnLabels() {
         return columnLabelIndices.keySet();
     }
+    
+    public List<String> columnLabelsList() { return Collections.unmodifiableList(columnLabels); }
```

Comment:
no docs, and can you make a list from `columnLabelIndices.keySet()` so you don't need a new member?